### ISSUE TITLE
Problem: Mascot images are cropped to a different ratio than specified image size

### ISSUE THREAD

**Issue by politas** [magicstone-dev/ecko#459]
The Site Settings page has the following:
Mascot image
Displayed on multiple pages. At least 293×205px recommended. When not set, falls back to default mascot.
Loading up an image with that ratio, eg:
![image](https://user-images.githubusercontent.com/2109967/204912369-72d0f467-ef04-4379-b43b-09aa3ea64453.png)
Leads to the server's About page showing that image with the bottom cropped out:
![image](https://user-images.githubusercontent.com/2109967/204912596-4abdd576-3709-4388-8315-fd5aa3285ade.png)

Either we should display the mascot image sized to fit within both X and Y dimensions, or we should provide the correct image ratio on the Site Settings page in order to prevent cropping.

**Comment by weex** [comment]
Confirmed. Pretty odd to be so specific about size and then crop it.

**Comment by politas** [comment]
Yes, and trying to use a different ratio image to make it fit in doesn't help. It scales the image to the desired _height_, then crops the result!
